packages/docusaurus-plugin-content-blog/src/index.ts

```diff
@@ -285,6 +287,9 @@ export default function pluginContentBlog(
               }),
               metadata: aliasedSource(pageMetadataPath),
             },
+            props: {
+              blogSiteDescription,
+            },
```

Comment:
you don't need that, you'll already receive metadata.blogDescription as props

Reply:
Hi @slorber , I found that to use  `blogDescription` in `metadata`, I still need to get it from `options` object and pass it to the `metadata` object here(about line 125):
```javascript
metadata: {
        permalink: blogPaginationPermalink(page),
        page: page + 1,
        postsPerPage,
        totalPages: numberOfPages,
        totalCount,
        previousPage: page !== 0 ? blogPaginationPermalink(page - 1) : null,
        nextPage:
          page < numberOfPages - 1
                ? blogPaginationPermalink(page + 1)
                : null,
        blogDescription: options.blogDescription,
}
```
Could you tell me if this is the case? Thank you!

Reply:
Yes, the blogDescription is a field on the plugin options

Reply:
OK! Thanks.